plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/BuildProgressSplitterPanelManager.kt

```diff
@@ -208,19 +209,14 @@ class BuildProgressSplitterPanelManager(private val project: Project) :
             }
         }
         val updatedStatuses = when (newState) {
-            TransformationStatus.CREATED, TransformationStatus.ACCEPTED -> {
+            TransformationStatus.CREATED, TransformationStatus.ACCEPTED, TransformationStatus.STARTED -> {
```

Comment:
Updated this to mark the upload as `DONE` when the status is CREATED, ACCEPTED, or STARTED